src/templates/import.js

```diff
@@ -684,6 +684,50 @@ module.exports = (router) => {
     });
   };
 
+  const tryToLoadComponent = (component, template) => {
+    const query = {
+      name: component.form,
+      deleted: {$eq: null}
+    };
+    return formio.resources.form.model.find(query).exec().then((results)=>{
```

Comment:
Since there is no filtering by 'project' in the query, it will fail or return a wrong form if:
1. We have form with this name in the project to which we import the template and inside another project (even if it is an another stage of the same project, it will have different projectId). It will throw an error 'Invalid template'.
2. We do not have a form with such name within the project we importing to,  but we have such a form inside another one. In this case it will import the form from another project to your project.